cli/cmd/integration_gar.go

```diff
@@ -0,0 +1,159 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func createGarIntegration() error {
+	questions := []*survey.Question{
+		{
+			Name:     "name",
+			Prompt:   &survey.Input{Message: "Name:"},
+			Validate: survey.Required,
+		},
+		{
+			Name: "domain",
+			Prompt: &survey.Select{
+				Message: "Registry Domain:",
+				Options: []string{
+					"northamerica-northeast1-docker.pkg.dev",
```

Comment:
Is this list subject to change?

Reply:
As GCP releases new regions, yes

Reply:
Yes. https://lacework.atlassian.net/browse/ALLY-596